templates/common/render/manifest.go

```diff
@@ -72,8 +72,8 @@ type writeManifestParams struct {
 	// in the destination directory.
 	outputHashes map[string][]byte
 
-	// The raw template source location, not necessarily canonical.
-	src string
+	// // The raw template source location, not necessarily canonical.
+	// src string
```

Comment:
Can't tell if this is intentionally commented out

Reply:
Oops, will fix. I'll make this change in the following PR in the series so I don't lose my precious PR approval.